nixos/modules/system/activation/bootspec.nix

```diff
@@ -79,9 +75,18 @@ let
 in
 {
   options.boot.bootspec = {
-    enable = lib.mkEnableOption (lib.mdDoc "Enable generation of RFC-0125 bootspec in $system/bootspec, e.g. /run/current-system/bootspec");
+    enable = lib.mkEnableOption (lib.mdDoc "the generation of RFC-0125 bootspec in $system/boot.json, e.g. /run/current-system/boot.json")
+      // { default = true; };
+    enableValidation = lib.mkEnableOption (
+      lib.mdDoc ''the validation of bootspec documents for each build.
```

Comment:
Isn't this already mdDoc? I see no such workarounds for other mkEnableOption calls.

Reply:
I can double check and get rid of it if it's the case.

Reply:
https://github.com/NixOS/nixpkgs/actions/runs/4764549907/jobs/8469269246
that's not the case AFAIK